app/src/test/kotlin/io/eugenethedev/taigamobile/repositories/SprintsRepositoryTest.kt

```diff
@@ -0,0 +1,192 @@
+package io.eugenethedev.taigamobile.repositories
+
+import io.eugenethedev.taigamobile.data.repositories.SprintsRepository
+import io.eugenethedev.taigamobile.domain.repositories.ISprintsRepository
+import io.eugenethedev.taigamobile.testdata.TestData
+import kotlinx.coroutines.runBlocking
+import org.junit.Test
+import java.time.LocalDate
+import kotlin.test.BeforeTest
+import kotlin.test.assertEquals
+
+class SprintsRepositoryTest: BaseRepositoryTest() {
+    lateinit var sprintsRepository : ISprintsRepository
+
+    @BeforeTest
+    fun setupSprintsRepositoryTest() {
+        sprintsRepository = SprintsRepository(mockTaigaApi, mockSession)
+    }
+
+    @Test
+    fun `test get user stories`() = runBlocking {
+        val userStories1 = sprintsRepository.getSprintUserStories(1)
+        val userStories2 = sprintsRepository.getSprintUserStories(2)
+
+        val testUserStories = TestData.projects[0].userstories.filter { it.sprint != null }
+
+        assertEquals(
+            expected = 1,
+            userStories1.size
+        )
+        assertEquals(
+            expected = testUserStories[0].name,
+            actual = userStories1[0].title
+        )
+
+        assertEquals(
+            expected = 1,
+            actual = userStories2.size
+        )
+        assertEquals(
+            expected = testUserStories[1].name,
+            actual = userStories2[0].title
+        )
+    }
+
+    @Test
+    fun `test get sprints`() = runBlocking {
+        val sprints = sprintsRepository.getSprints(1)
+
+        assertEquals(
+            expected = TestData.projects[0].sprints.size,
+            actual = sprints.size
+        )
+        assertEquals(
+            expected = TestData.projects[0].sprints.map { it.name },
+            actual = sprints
+                .sortedBy {it.name} //Upload in vice versa order, therefore need this
```

Comment:
I think it would be clearer if you sort both lists. Or you can just use sets, because we have convention about sprint names (refer to the comment above)